environment/visibility.csv

```diff
@@ -182,7 +182,8 @@ TGHO,"2m pole ~ 1m to south of mark, otherwise no obstructions.",2008-12-22T00:0
 TGHR,No obstructions.,2006-04-11T00:00:00Z,9999-01-01T00:00:00Z
 TGOH,clear all round except 7 deg hill at SE. Masts to 5m at 070mag,2006-04-01T00:00:00Z,9999-01-01T00:00:00Z
 TGOH,vegetation growing around the antenna imapcting time series,2014-11-01T00:00:00Z,2015-03-31T23:59:59Z
```

Comment:
can we please change this to:
`TGOH,vegetation growing around antenna. Obstructions at 7deg hill at SE and 5m mast at 070mag,2006-04-01T00:00:00Z,9999-01-01T00:00:00Z`